packages/react-dev-utils/fileSizeReporter.js

```diff
@@ -0,0 +1,85 @@
+var fs = require('fs');
```

Comment:
Mind renaming it to `FileSizeReporter` since it's not exporting a single function?